### PR TITLE
Doc reorg

### PR DESCRIPTION
**Pull request by Tyrosine22**
Submitting the re-orged docs, sans the unfinished PSM documentation. (It'll be included in a subsequent PR.)

### CODE REVIEW

**Comment by dckc** [comment]
We usually get cloudflare previews. I don't see one here. I wonder what's up.

**Comment by Tyrosine22** [comment]
If you're wanting to preview what the docs will look like on the web site,
you can locally clone the docreorg branch & follow the instructions in
CONTRIBUTING.md in the repo root directory to create a local doc build.
It's pretty easy to set up local doc builds.

-Mark

**Comment by dckc** [comment]
yes, I have a local build now. Still... I wonder why the cloudflare preview isn't working.

**Comment by Tyrosine22** [comment]
I'll check out cloudflare's documentation... maybe something I did in the
repo disabled it somehow.

File: main/conceptual/dapps/README.md
Comment (dckc) [inline main/conceptual/dapps/README.md:1]: Ugh. The github diff makes it look like this is new too, but it looks similar to https://docs.agoric.com/dapps/

File: main/conceptual/dapps/starting-multiuser-dapps.md
Comment (dckc) [inline main/conceptual/dapps/starting-multiuser-dapps.md:1]: another case of the github diff failing us somehow

File: main/conceptual/ertp/README.md
Comment (dckc) [inline main/conceptual/ertp/README.md:22]: Note to self: check that CONTRIBUTING.md notes the preference for relative links.

File: main/conceptual/ertp/README.md
Comment (dckc) [inline main/conceptual/ertp/README.md:231]: hm. periods for things that are not sentences?

File: main/conceptual/ertp/issuers-and-mints.md
Comment (dckc) [inline main/conceptual/ertp/issuers-and-mints.md:20]: you proof-read _all_ of the text?! remarkable!

File: main/conceptual/ertp/purses-and-payments.md
Comment (dckc) [inline main/conceptual/ertp/purses-and-payments.md:151]: This seems a little odd, organizationally. But it's no worse than it was before...

File: main/conceptual/getting-started/deploying.md
Comment (dckc) [inline main/conceptual/getting-started/deploying.md:3]: this link isn't relative? Why change the others to be relative, then?

File: main/conceptual/getting-started/start-a-project.md
Comment (dckc) [inline main/conceptual/getting-started/start-a-project.md:9]: That emphasis seemed reasonably important, based on feedback we've gotten from people using the docs.

File: main/conceptual/getting-started/start-a-project.md
Comment (dckc) [inline main/conceptual/getting-started/start-a-project.md:24]: "tildot" means `~.`; if you prefer not to use / introduce that term, the replacement isn't just "tilde" but something like "tilde-dot".

File: main/conceptual/js-programming/eventual-send.md
Comment (dckc) [inline main/conceptual/js-programming/eventual-send.md:1]: hm. why remove the `E()` formatting?

File: main/conceptual/js-programming/eventual-send.md
Comment (dckc) [inline main/conceptual/js-programming/eventual-send.md:36]: no, the message doesn't contain a method (that would be a function). It contains a method _name_; that is: a string.

And what gets marshaled is a list... but maybe that part is clear enough either way.

File: main/conceptual/js-programming/far.md
Comment (dckc) [inline main/conceptual/js-programming/far.md:2]: So we're getting rid of code markup in headings? Does it make sense to keep the ()s, then?

File: main/conceptual/js-programming/hardened-js.md
Comment (dckc) [inline main/conceptual/js-programming/hardened-js.md:14]: you're no fun. :)

File: main/conceptual/js-programming/hardened-js.md
Comment (dckc) [inline main/conceptual/js-programming/hardened-js.md:52]: It's capitalized OCap / OCaps in places like https://agoric.com/wp-content/uploads/2022/08/Agoric-White-Paper-v1.1.pdf

File: main/conceptual/js-programming/hardened-js.md
Comment (dckc) [inline main/conceptual/js-programming/hardened-js.md:74]: The big C in OCap is on purpose; if you really want to change it, please get review from MarkM.

File: main/conceptual/js-programming/hardened-js.md
Comment (dckc) [inline main/conceptual/js-programming/hardened-js.md:221]: The `caller` example seems particularly relevant and illustrative. `arguments` might be an even better one.

File: main/conceptual/js-programming/notifiers.md
Comment (dckc) [inline main/conceptual/js-programming/notifiers.md:70]: should "If" be capitalized here?

File: main/conceptual/js-programming/notifiers.md
Comment (dckc) [inline main/conceptual/js-programming/notifiers.md:156]: can we get rid of "Click on ..." while were at it?

[Avoid talking about mechanisms](https://www.w3.org/Provider/Style/NoMechanics.html)

File: main/conceptual/zoe/contracts/README.md
Comment (dckc) [inline main/conceptual/zoe/contracts/README.md:1]: ugh. another "file looks new" diff problem

File: main/conceptual/zoe/contracts/vault.md
Comment (dckc) [inline main/conceptual/zoe/contracts/vault.md:123]: Taking these "future directions" lists out makes sense to me... @Chris-Hibbert do you agree?

If they're worth tracking, issues in the agoric-sdk repo make more sense, to me.

File: main/reference/ertp-api/purse.md
Comment (dckc) [inline main/reference/ertp-api/purse.md:9]: why drop `payments` here?

File: main/reference/ertp-api/purse.md
Comment (dckc) [inline main/reference/ertp-api/purse.md:42]: `purse` rather than **purse**?

**Review by dckc — CHANGES_REQUESTED** [review]
remarkable proofreading work!

The redirects seem to need `.html` rather than `.md`.

I feel kinda strongly that `/conceptual/` is not an improvement over `/guides/`.

One change in marshal went beyond editorial.

I think the rest of my comments are not critical.

File: main/conceptual/zoe/contracts/vault.md
Comment (Chris-Hibbert) [reply main/conceptual/zoe/contracts/vault.md:123]: Yes, makes sense. while these are actually relatively high-priority product directions, it's probably a bad idea to talk about product directions in the write-up for each component. In most places, they'll rapidly become stale.

File: main/.vuepress/themeConfig/nav.js
Comment (Tyrosine22) [reply main/.vuepress/themeConfig/nav.js:99]: In the current docs, the Dynamic IBC link pointed to the Network API repo documentation. But there's a link on the REPL API page which points to a markdown file within the doc repo which contains literally the exact same content. So either the Dynamic IBC link was pointing at the wrong thing, or it was a duplicate link. I was going to investigate which when I finished revising the docs after PSM is published.

File: main/.vuepress/enhanceApp.js
Comment (Tyrosine22) [reply main/.vuepress/enhanceApp.js:21]: The link checking tool doesn't check redirects; only internal links. What's more, I discovered several broken links which the tool wasn't catching. When I get some time, I'm planning on either fixing our link checking tool or simply writing one of my own; it's not hard.

File: main/.vuepress/themeConfig/nav.js
Comment (Tyrosine22) [reply main/.vuepress/themeConfig/nav.js:30]: You're right about "components", but let's leave it alone for now. I'm not sure if it's best to break the non-components into a different bucket, or if there's a word that encompasses all the "things", or what. I want to give it some thought.... and components is good enough for now.

As for Agoric-cli, the "reference" section is really "API reference", so the CLI commands don't belong in that bucket. At least, that's how I use docs; I use API reference docs when I have an IDE open & am coding, but I want all the other content separated from the API reference. (I'm basically following the MSFT style guide here, but that's because that's what I'm used to.)

File: main/conceptual/README.md
Comment (Tyrosine22) [reply main/conceptual/README.md:1]: Yeah, it was supposed to be internal-only. If we simply delete that README.md, users get a 404 error if they go to /conceptual/, so I instead changed its text to your suggestion.

File: main/conceptual/ertp/README.md
Comment (Tyrosine22) [reply main/conceptual/ertp/README.md:231]: I'm following the MSFT guide for those because I'm used to it & it "looks right" to me. The rule is: descriptions like that, even when they're sentence fragments, start with a capital letter and end with a period.

File: main/.vuepress/themeConfig/nav.js
Comment (dckc) [reply main/.vuepress/themeConfig/nav.js:30]: fair enough.

File: main/.vuepress/themeConfig/nav.js
Comment (dckc) [reply main/.vuepress/themeConfig/nav.js:99]: ah. so we're not really dropping anything.

File: main/conceptual/ertp/purses-and-payments.md
Comment (Tyrosine22) [reply main/conceptual/ertp/purses-and-payments.md:151]: I modified the formatting of the section header to conform to the standard style I'm using for all the section headers.

File: main/conceptual/getting-started/deploying.md
Comment (Tyrosine22) [reply main/conceptual/getting-started/deploying.md:3]: Well, it's semi-relative; it's not an absolute link. (i.e., https://docs.agoric.com/getting-started/before-using-agoric.html) I found both kinds of relative links already in the docs, (i.e., "/reference/repl/" and "../../reference/repl/") and they're both fine as far I'm concerned.

File: main/conceptual/getting-started/start-a-project.md
Comment (Tyrosine22) [reply main/conceptual/getting-started/start-a-project.md:9]: K. I put the bold formatting back in. 

(I don't like how this page reads, but it's not worth spending the time right now to revise it. I'll come back to this topic sometime in the future.)

File: main/conceptual/getting-started/start-a-project.md
Comment (Tyrosine22) [reply main/conceptual/getting-started/start-a-project.md:24]: ! I've never seen the word "tildot" before; I thought it was a typo. I reverted the text back to "tildot"

File: main/conceptual/js-programming/eventual-send.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/eventual-send.md:1]: I really don't like how the docs use the code snippet formatting (`E()`) for atomic programming items (e.g., function names, object names, etc.) I prefer to reserve that formatting for entire programming ideas (e.g., a proper function call: `E(zoe).getBrands(myInstance)`), and I used bold for atomic programming items. I find it much more readable & usable that way. Also, I prefer to make the first occurrence of an atomic item a link to the reference docs where the item is described.
 But then Rowland's request for the PSM docs reshuffled my priorities, and I didn't go through the docs & universally fix the formatting. I'll do so sometime in the future. (I should prolly change that formatting back to `E()` for consistency's sake, but I'm being lazy.)

File: main/conceptual/js-programming/far.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/far.md:2]: I was still working out what formatting looked right when I got interrupted by Rowland's request. (I really didn't like the code markup in the heading; it wasn't very readable & looked weird.)

So... yeah. I'll come back to this in the near future & apply a consistent formatting throughout the docs.

File: main/conceptual/getting-started/start-a-project.md
Comment (dckc) [reply main/conceptual/getting-started/start-a-project.md:24]: I think we made it up as part of the discussion of this JavaScript syntax extension.

File: main/conceptual/js-programming/eventual-send.md
Comment (dckc) [reply main/conceptual/js-programming/eventual-send.md:1]: ok.

File: main/conceptual/js-programming/eventual-send.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/eventual-send.md:36]: Right you are. I changed the text to "A message consisting of the method name install with the bundle argument marshaled to a flat string and queued for delivery to the vat that zoe comes from." It's still not quite correct, since it gets marshaled to a series of strings, not "a flat string", exactly. But it's better than what I had, and no worse than what was there previously, so it's good enough for now.

File: main/conceptual/js-programming/hardened-js.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/hardened-js.md:52]: This is probably going to be more information than you want, but....
I looked online to see how OCap's capitalization is usually handled, and there doesn't seem to be a consensus. When it's spelled out, it's usually (but not always) not hyphenated or capitalized, but it gets capitalized here because it's a section header. As for "OCap", its capitalization is all over the place, so I generally left it as whatever we had it as. I really should decide on our official style guide for it, and then make the docs consistent throughout, but I have higher priority stuff to worry about. So for now, we're going with "willy-nilly capitalization".

File: main/conceptual/js-programming/hardened-js.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/hardened-js.md:74]: ...and here I broke the rule of "leave it alone" that I laid out in the previous comment.  Lovely. I changed it back to "OCaps".

File: main/conceptual/js-programming/hardened-js.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/hardened-js.md:221]: I don't know what "no caller" means. Is there a specific example elsewhere in the page that I'm not seeing? In your defense, I haven't read the entire page carefully. In my defense, readers will also often not read the entire page carefully. 
For now, I'll bow to your superior knowledge of the audience; I've reverted the text so that "(no caller etc.)" appears. But when I revisit this topic for proper revision, I'll ping you to get clear what you're getting at & hopefully clarify the text accordingly.

File: main/conceptual/js-programming/notifiers.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/notifiers.md:70]: Yes. The "thing" within the parenthesis is a proper sentence, so it should follow the rules of a normal sentence. That's also why I added a period to it inside the parenthesis.

File: main/conceptual/js-programming/notifiers.md
Comment (Tyrosine22) [reply main/conceptual/js-programming/notifiers.md:156]: Good catch. The existing docs talk a lot about mechanisms, and I tried to get rid of those. I missed this one. (That sentence is now removed.)

File: main/conceptual/zoe/contracts/vault.md
Comment (Tyrosine22) [reply main/conceptual/zoe/contracts/vault.md:123]: I thought I had removed that; it doesn't exist in my local copy. I'm not sure what happened. In theory, when I push my additional revisions to the cloud, this roadmap will go away.

File: main/reference/ertp-api/purse.md
Comment (Tyrosine22) [reply main/reference/ertp-api/purse.md:9]: Because this topic is discussing purses; including a detail like that about payments isn't very helpful. Such information should instead also appear on the payments topic.  
Upon inspection, I noticed that payments did not have that info, so I added it to the payments topic as well.

File: main/reference/ertp-api/purse.md
Comment (Tyrosine22) [reply main/reference/ertp-api/purse.md:42]: I'm not sure what revision you're suggesting; the text had `purse` rather than **purse**. However, I noticed that I revised this API entry w/out revising any other entries. So I reverted this entry back to the previous format until I have a chance to update all the API entries to whatever new format we decide upon.

File: main/conceptual/js-programming/hardened-js.md
Comment (dckc) [reply main/conceptual/js-programming/hardened-js.md]: "(no caller etc.)" refers to `.caller` in the red part of the diagram just previous:

Hardened JavaScript _ Documentation


We could supplement with a link to [caller](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/caller) in MDN perhaps.

File: main/conceptual/js-programming/notifiers.md
Comment (dckc) [reply main/conceptual/js-programming/notifiers.md]: The 1st word in the sentence is "e.g." though, yes?

FWIW:
 - [orthography \- When a sentence starts with "e\.g\.", should the e be capitalized? \- English Language & Usage Stack Exchange](https://english.stackexchange.com/questions/28172/when-a-sentence-starts-with-e-g-should-the-e-be-capitalized)

File: main/conceptual/zoe/contracts/vault.md
Comment (dckc) [reply main/conceptual/zoe/contracts/vault.md]: You did remove it; that's why it's red. I just wanted to confirm with Chris that it _should_ be removed.

**Review by dckc — APPROVED** [review]
good stuff.

**Pull request merged**
